lib/erlef_web/templates/page/fellows.html.eex

```diff
@@ -0,0 +1,44 @@
+<div id="fellows" class="container">
+  <div>
+    <h1 data-toc-skip="true" style="margin-left:1rem;">Fellows</h1>
+    <p style="margin-left:1rem;">
+    What is a fellow? Those whom have made extraordinary contributions to our community. 
```

Comment:
Need input here. Might to broad or not broad enough.

Reply:
This is good enough. There might be some value in explaining how they're chosen, but that could also just be a link to the fellows WG

Reply:
Yeah, agreed, good enough.